scipy/stats/tests/test_resampling.py

```diff
@@ -945,8 +945,9 @@ def test_alternative(self, alternative, xp):
 
         res = monte_carlo_test(x, rvs, statistic, alternative=alternative)
 
-        xp_assert_close(res.statistic, xp.asarray(ref.statistic))
-        xp_assert_close(res.pvalue, xp.asarray(ref.pvalue), atol=self.atol)
+        xp_assert_close(res.statistic, xp.asarray(ref.statistic), check_0d=False)
+        xp_assert_close(res.pvalue, xp.asarray(ref.pvalue), atol=self.atol,
+                        check_0d=False)
```

Comment:
why not `_array_api_no_0d` in this file?

Reply:
I've changed that over for you now, so you don't need to wonder in stats if anything might have slipped through.